Server/Utilities/JSONUtils.m

```diff
@@ -37,11 +37,41 @@ + (NSArray *)elementTypes {
             sortedArrayUsingSelector:@selector(localizedCaseInsensitiveCompare:)];
 }
 
++ (CBXVisibilityResult *)visibilityResult: (XCUIHitPointResult*)hitPoint {
+    CBXVisibilityResult *cbxResult = [CBXVisibilityResult new];
+    CGPoint intermediate;
+    XCUIHitPointResult *result = hitPoint;
```

Comment:
Why do we need separate variable result? We already have an hitPoint variable which came from method parameters.

Reply:
Previously this method was described in XCUIElement+VisibilityResult.m, but since XCElementSnapshot framework was hidden, I decided to implement this method here.